src/Migration/Sources/Appwrite.php

```
@@ -1996,6 +2035,99 @@ private function exportSiteDeploymentData(Site $site, array $deployment): void
         }
     }
 
+    /**
+     * @param array<string> $resources
+     * @param array<string, int> $report
+     * @param array<string, array<string>> $resourceIds
+     */
+    private function reportIntegrations(array $resources, array &$report, array $resourceIds = []): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resourceIds'. (undefined)

(UnusedFormalParameter)